lib/clangimport.cpp

```diff
@@ -509,7 +509,7 @@ void clangimport::AstNode::setLocations(TokenList *tokenList, int file, int line
         } else if (ext[0] == '<') {
             const std::string::size_type colon = ext.find(':');
             if (colon != std::string::npos) {
-                const bool windowsPath = colon == 2 && ext.size() > 4 && ext[3] == '\\';
+                const bool windowsPath = colon == 2 && ext.size() > 3 && ext[2] == ':';
```

Comment:
Some logic for this already exists in `Path::isAbsolute()` and maybe other places as well. We could put this in a helper function later on and share the code.

Reply:
I don't think that would work, since we are already inside a larger string here. We only need to know if there is a second colon.